src/ui/fullscreenhandlerbridge.ts

```diff
@@ -11,16 +11,28 @@ const FullscreenHandlerModule = NativeModules.FullscreenHandlerModule;
 export class FullscreenHandlerBridge {
   readonly nativeId: string;
   fullscreenHandler?: FullscreenHandler;
+  isDestroyed: boolean;
 
   constructor(nativeId?: string) {
     this.nativeId = nativeId ?? Uuid.generate();
+    this.isDestroyed = false;
     BatchedBridge.registerCallableModule(
-      `bmFullscreenBridge-${this.nativeId}`,
+      `FullscreenBridge-${this.nativeId}`,
       this
     );
     FullscreenHandlerModule.registerHandler(this.nativeId);
   }
 
+  /**
+   * Destroys the native FullscreenHandler
+   */
+  destroy() {
+    if (!this.isDestroyed) {
+      FullscreenHandlerModule.destroy(this.nativeId);
+      this.isDestroyed = true;
+    }
+  }
+
```

Comment:
This was added to clean up fullscreen bridge once the player view gets deallocated.